### PR TITLE
Include Windows 10 in the Platform enum.

### LINKED ISSUE #669: Include WIN10 in Platform enum
As it will be released soon, it has become necessary to include Windows 10 in Platform enum.

### PR DESCRIPTION
**Pull request by ngsankha**
Fixes #669.

I have already signed the CLA.

### CODE REVIEW

File: java/client/src/org/openqa/selenium/Platform.java [hunk 1]
```diff
@@ -72,6 +72,13 @@ public Platform family() {
     }
   },
 
+  WIN10("windows 10", "win10") {
```
Comment (ddavison) [inline java/client/src/org/openqa/selenium/Platform.java]: we'll also have to think about adding "Windows Server 2014" or whatever they come up with later.

Comment (sankha93) [inline java/client/src/org/openqa/selenium/Platform.java]: @ddavison Should I add it, or is this commit good for merging?

Comment (ddavison) [inline java/client/src/org/openqa/selenium/Platform.java]: it is good for merging at this point, but i'm awaiting others.  it's likely that we won't merge this until it actually comes out officially.

**Comment by santiycr** [comment]
LGTM. Ready to merge?

**Comment by ddavison** [comment]
assuming that these *are* the values that the platform returns, then yep!

**Comment by santiycr** [comment]
the capabilities returned back are actually:
```json
"platform": "Windows NT"
```

But that's the case for Win 7, Win 8 and even Win XP, as far as I can tell :/

**Comment by AndyUM357** [comment]
Is there a reason this is feature is taking so long to merge?  I show os.name: 'Windows 10', os.version: '10.0'  The new code should work, although a test or two added to java/client/test/org/openqa/selenium/PlatformTest.java wouldn't hurt, but this should be good to go.  Please merge :)

**Comment by ddavison** [comment]
merged in as 1c37c7231b1bf8fbb979c04ef367cb51e5755965. thanks!

**Pull request closed without merging**